common/web/keyboard-processor/src/text/defaultRules.ts

```diff
@@ -1,32 +1,28 @@
-// TODO:  Move to separate folder:  'codes'
-// We should start splitting off code needed by keyboards even without a KeyboardProcessor active.
-// There's an upcoming `/common/web/types` package that 'codes' and 'keyboards' may fit well within.
-
-import { ModifierKeyConstants} from '@keymanapp/common-types';
-import Codes from "./codes.js";
-import type KeyEvent from "./keyEvent.js";
-import type OutputTarget from "./outputTarget.js";
-
-// The only members referenced are to produce warning and error logs.  A little abstraction
-// via an optional 'logger' interface can maintain it while facilitating a the split alluded
-// to above.
-//
-// Alternatively, we could just... not take in the parameter at all, which'd also facilitate
-// the future modularization effort.
-import RuleBehavior from "./ruleBehavior.js";
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * Implementation of default rules
+ */
+
+import { ModifierKeyConstants } from '@keymanapp/common-types';
+import Codes from './codes.js';
+import type KeyEvent from './keyEvent.js';
+import type OutputTarget  from './outputTarget.js';
```

Comment:
I just noticed a possibility for other improvements when reviewing #12131.  It fits better thematically here, due to relocating an extra file, but I'm fine with it being done within either PR.

https://github.com/keymanapp/keyman/pull/12131#discussion_r1716246116
> This part [`OutputTarget` / `text/outputTarget.ts`] belongs in `js-processor` if at all possible.  It's one of the text-processing / rule-processing interfaces.
>
> I realize that `DefaultRules` _currently_ has references to `OutputTarget`, but upon inspection, I see that they're essentially dead on this level. 



https://github.com/keymanapp/keyman/pull/12131#discussion_r1716247453

> Something I noticed, which gives us the opportunity to simplify here:  the `applyCommand` method isn't even called here.  I think it's overridden and used within `app/browser` (or maybe `engine/main`), but since it's not actually referenced for the base class, it should be safe to delete.

> Deleting `applyCommand` here will allow us to drop the references to `OutputTarget`... and I think this is the only `engine/keyboard` reference to `OutputTarget` within the package, allowing us to them move that file to `js-processor`.

Reply:
I realize we do have references to the type on `Keyboard` - largely due to `gs`, `gn`, and `gpk` declarations on `KeyboardObject` within keyboard.ts.  There's not really a cleaner way around this that would also preserve typing.

Reply:
Will be done in #12131.